### ISSUE TITLE
Classification accuracy ?

### ISSUE THREAD

**Issue by capilano** [davidsandberg/facenet#1048]
What would the classification accuracy score look like on the training set? Let's say if we had around 5-6 million images and around 90K classes(Assuming the training loss objective is some variant of softmax loss)
 Is something around 0.55 (I have around 90K classes(unique faces) and 6 million images), a reasonable score? I am using cosine loss.

**Comment by ani16** [comment]
Hi

I am also not getting good classification accuracy. Infact I trained Model with few images from LFW dataset using pre trained model of LFW dataset and test for few LFW images only. Still failes to give good accuracy.
